deploy/olm/generate-rook-csv.sh

```diff
@@ -213,6 +213,8 @@ function generate_package() {
 function apply_rook_op_img() {
     "${YQ_CMD_WRITE[@]}" "$CSV_FILE_NAME" metadata.annotations.containerImage "$ROOK_OP_VERSION"
     "${YQ_CMD_WRITE[@]}" "$CSV_FILE_NAME" spec.install.spec.deployments[0].spec.template.spec.containers[0].image "$ROOK_OP_VERSION"
+    "${YQ_CMD_WRITE[@]}" "$CSV_FILE_NAME" spec.install.spec.deployments[0].spec.template.spec.containers[0].env[6].value "true"
```

Comment:
The `env[6]` could be brittle if another env var is added or removed. Instead of replacing the value like this, I wonder if it would be simpler to leave the env var out of the operator-openshift.yaml, and append the env var here with yq.

Reply:
this is something I tried most of the time today to make more generalize and I was able to do it is possible wit yq 4.x but currently in rook csv generation its hard requirements 3.x